### PR TITLE
TC 01.1.1_03 Header has title What's New

### PR DESCRIPTION
**Pull request by NataliaStarkova**

### CODE REVIEW

File: tests/header.spec.js [hunk 1]
```diff
@@ -95,5 +95,25 @@ test.describe('header', () => {
...
+
+  test('TC 01.1.1_05 <Header/Header logo> Click title Women', async ({page}) => {
+    await page.getByRole('menuitem', {name: ' Women'}).click()
+    await expect(page).toHaveURL('https://magento.softwaretestingboard.com/women.html')
+  })
+
+  test('TC 01.1.1_06 <Header/Header logo> Click title Men', async ({page}) => {
+    await page.getByRole('menuitem', {name: ' Men'}).click()
+    await expect(page).toHaveURL('https://magento.softwaretestingboard.com/men.html')
+})
```
Comment (S-Lanchik) [inline tests/header.spec.js:117]: One PR should have one test.

Comment (S-Lanchik) [reply tests/header.spec.js:117]: Add link to Trello card

Comment (NataliaStarkova) [reply tests/header.spec.js]: I'm ready))

**Comment by NataliaStarkova** [comment]
https://trello.com/c/bepv9Tbb/254-tc-011103-header-has-title-whats-new

File: tests/header.spec.js (hunk 1 above)
Comment (S-Lanchik) [reply tests/header.spec.js]: Please add trello card link in the comment section

**Comment by S-Lanchik** [comment]
Also add PR link to Trello

**Comment by S-Lanchik** [comment]
Also "What's New" is not a header, it is one of the navigation option, so rename your test and PR.

**Review by BabintsevaS — APPROVED** [review]

**Pull request closed without merging**